enterprise/coderd/license/license.go

```diff
@@ -52,6 +52,13 @@ func Entitlements(
 		return entitlements, xerrors.Errorf("query active user count: %w", err)
 	}
 
+	// always shows active user count regardless of license
```

Comment:
nit: I'd like to see a unit test covering this change.

Reply:
BTW if you find some troubles with implementing it, feel free to merge this PR first, so you're unblocked.

Reply:
I just added a test covering it! :) thx!

Reply:
https://github.com/coder/coder/pull/7210/commits/0322172496a11647cfc753f3ca0e14c66681ad1f